src/component/container/Footer/Footer.tsx

```diff
@@ -222,12 +220,13 @@ export class Footer extends React.Component<FooterProps, FooterState> {
           <span>{t('MousePositionLabel')}: </span>
           <div id="mouse-position"/>
         </div>
-        <div className="imprint footer-element">
-          <a
-            href={imprintLink ? imprintLink : 'https://www.terrestris.de/en/impressum'}
-          >
-            {imprintText ? imprintText : `${t('Imprint.title')} / ${t('Imprint.privacypolicy')}`}</a>
-        </div>
+        {imprint &&
+          <div className="footer footer-element">
+            {imprint ? <div dangerouslySetInnerHTML={{ __html: imprint }} /> :
```

Comment:
I think this check (`imprint ? ...`) is unnecessary here since the whole div-block won't be rendered if no imprint is provided (s. L223)

Reply:
Good catch, thanks.